velox/connectors/hive/HiveConnectorSplit.h

```diff
@@ -82,8 +82,7 @@ struct HiveConnectorSplit : public connector::ConnectorSplit {
   }
 
   std::string getFileName() const override {
-    auto i = filePath.rfind('/');
-    return i == std::string::npos ? filePath : filePath.substr(i + 1);
+    return filePath;
```

Comment:
This is the required behavior for other systems and cannot be changed.   Can you add `getFullFileName` and depends on that instead?

Reply:
got! update @Yuhta